components/low/lift.py

```diff
@@ -12,24 +12,23 @@ class Lift:
     lift_drive_right: ctre.WPI_VictorSPX
     lift_prox_front: wpilib.DigitalInput
     lift_prox_back: wpilib.DigitalInput
-    lift_limit_front_top: wpilib.DigitalInput
-    lift_limit_front_bottom: wpilib.DigitalInput
-    lift_limit_rear_top: wpilib.DigitalInput
-    lift_limit_rear_bottom: wpilib.DigitalInput
-    
+    lift_limit_front: wpilib.DigitalInput
+    lift_limit_back: wpilib.DigitalInput
```

Comment:
There's a halleffect here too.

Reply:
These two "limits" are just the two halleffects

Reply:
I see.